src/libraries/System.Net.Security/tests/StressTests/SslStress/SslStress.csproj

```diff
@@ -8,7 +8,7 @@
     <Compile Include="..\..\..\..\Common\tests\System\IO\Compression\CRC.cs" Link="Utils\CRC.cs" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="System.CommandLine.Experimental" Version="0.3.0-alpha.19405.1" />
+    <PackageReference Include="System.CommandLine.Experimental" Version="0.3.0-alpha.19577.1" />
```

Comment:
It seems like these two StressTests can share the package version in their `Directory.Build.props`?